openpype/hosts/blender/api/lib.py

```diff
@@ -116,28 +119,31 @@ def get_user_links(
 
     # Get all datablocks from blend file
     all_datablocks = set()
-    for datacol in dir(bpy.data):
+    for datacol_name in dir(bpy.data):
+        datacol = getattr(bpy.data, datacol_name)
         if not isinstance(
-            getattr(bpy.data, datacol),
+            datacol,
             bpy.types.bpy_prop_collection,
-        ):
+        ) and datacol not in {bpy.data.collections, bpy.data.objects}:
```

Comment:
Is this safer than checking for type? I think a type check would run faster, but if there's an advantage in the way you did it it's fine for me.

Reply:
The thing is, `bpy.data.collections` and `bpy.data.objects` are same type (`bpy_prop_collection`), if I want to type check, I'd have to access the first item of each list and compare, which is much slower than comparing lists reference directly.